### PR TITLE
extensions: create the gnome-platform directory

### PR DESCRIPTION
**Pull request by sergiusens**
This is the target directory for the gnome "platform" content interface plug.

Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

- [ ] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] Have you successfully run `./runtests.sh static`?
- [ ] Have you successfully run `./runtests.sh tests/unit`?

-----

### CODE REVIEW

File: extensions/desktop/gnome/Makefile [hunk 1]
```diff
@@ -17,14 +17,15 @@ clean:
 
 $(DEST_LAUNCHER):
 	@cat $(SRC_DIR)/init >> $(DEST_LAUNCHER)
-	# tail -n +2 to remove the shebang
+# tail -n +2 to remove the shebang
```
Comment (cjp256) [inline extensions/desktop/gnome/Makefile]: lost the tab?

File: extensions/desktop/gnome/Makefile [hunk 2]
```diff
@@ -17,14 +17,15 @@ clean:
...
 	@tail -n +2 $(SRC_DIR)/desktop-exports >> $(DEST_LAUNCHER)
 	@tail -n +2 $(SRC_DIR)/launcher-specific >> $(DEST_LAUNCHER)
 	@tail -n +2 $(SRC_DIR)/mark-and-exec >> $(DEST_LAUNCHER)
 	gcc -Wall -O2 -o $(BINDTEXTDOMAIN) -fPIC -shared $(SRC_DIR)/../src/bindtextdomain.c -ldl
-		
+
 install: $(DEST_LAUNCHER)
 	install -d $(DATA_DIR)
+	install -d $(DATA_DIR)/gnome-platform
```
Comment (cjp256) [inline extensions/desktop/gnome/Makefile]: I don't think this goes under data-dir?

File: extensions/desktop/gnome/Makefile (hunk 1 above)
Comment (sergiusens) [reply extensions/desktop/gnome/Makefile]: the tab showed as a syntax error on emacs so I removed it, yes.

File: extensions/desktop/gnome/Makefile (hunk 2 above)
Comment (sergiusens) [reply extensions/desktop/gnome/Makefile]: good catch!

File: extensions/desktop/gnome/Makefile (hunk 1 above)
Comment (cjp256) [reply extensions/desktop/gnome/Makefile]: Now it looks like a syntax error :P I'm not sure the comment is really necessary anyhow.

If these scripts are not intended to be used as-is, perhaps it makes sense to just drop the shebangs from the source?

File: extensions/desktop/gnome/Makefile [hunk 3]
```diff
@@ -17,13 +18,14 @@ clean:
 
 $(DEST_LAUNCHER):
 	@cat $(SRC_DIR)/init >> $(DEST_LAUNCHER)
-	# tail -n +2 to remove the shebang
+# tail -n +2 to remove the shebang
```
Comment (zyga) [inline extensions/desktop/gnome/Makefile]: Technically the tab is not a syntax error here. Though this is not the essential part.

Comment (sergiusens) [reply extensions/desktop/gnome/Makefile]: Oh, it is not, as it was working, just visual tolerance on my side

**Pull request merged**